es6/README.md

```diff
@@ -0,0 +1,403 @@
+# Linting
+
+Included is a default ESLint configuration, `mobify-es6.yml` written for ESLint 3.x. It requires the additional modules:
+
+- `eslint-plugin-import`
+- `eslint-import-resolver-webpack`
```

Comment:
Yeah, I guess minimizing unnecessary npm module installs sounds better in this case.

Do we also need `eslint-plugin-react`? That was included in the scaffold PR